### PR TITLE
fix: time display

### LINKED ISSUE #125: last-update tag stays in "0 seconds ago"
from @Septias:
> About the 0 secs: I was also thinking to rework this to show everything smaller than 1m as 'just now' and then update minutes etc.

from @hpk42:
> (for me the time does not update by itself which i would expect from a "duration ago" viewer)

### PR DESCRIPTION
**Pull request by Septias**
closes #125

### CODE REVIEW

**Comment by adbenitez** [comment]
"less than a minute" is a bit long, what about "recently" or "now"?

**Comment by adbenitez** [comment]
deployed this PR, but the "less than a minute" label is shown every time you open the app, it is not really remembering when was the last time of the sync but starting to count from the point the app is opened

File: frontend/src/entry-points/shop.tsx [hunk 1]
```diff
@@ -270,7 +268,7 @@ const Shop: Component = () => {
             <div class="rounded-xl bg-gray-100 p-2 unimportant text-gray-500">
               <Show when={isUpdating()} fallback={
                 <button class="flex items-center gap-2" onclick={handleUpdate}>
-                  <span>{formatDuration(timeSinceLastUpdate(), { delimiter: ',' }).split(',')[0] || '0 sec'} ago</span>
+                  <span>{formatDistance(lastUpdate(), dateNow())}</span>
```
[the remark this answers is not in the document]
Comment (Septias) [reply frontend/src/entry-points/shop.tsx:271]: will not be reactive this way

**Comment by hpk42** [comment]
let's merge this and iterate on it as needed.

**Pull request merged**